runelite-client/src/main/java/net/runelite/client/plugins/fishing/FishingPlugin.java

```diff
@@ -335,4 +347,40 @@ public void onVarbitChanged(VarbitChanged event)
 			trawlerNotificationSent = false;
 		}
 	}
+
+	@Subscribe
+	public void onWidgetLoaded(WidgetLoaded event)
+	{
+		if (event.getGroupId() == WidgetID.FISHING_TRAWLER_GROUP_ID)
+		{
+			startTime = Instant.now();
+		}
+	}
+
+	/**
+     * Changes the Fishing Trawler timer widget from minutes to minutes and seconds
+     *
+     */
+	private void updateTrawlerTimer()
+	{
+		int regionID = client.getLocalPlayer().getWorldLocation().getRegionID();
+
+		if (regionID != TRAWLER_SHIP_REGION_NORMAL && regionID != TRAWLER_SHIP_REGION_SINKING || startTime == null)
+		{
+			return;
+		}
+
+		Widget trawlerTimerWidget = client.getWidget(WidgetID.FISHING_TRAWLER_GROUP_ID, 37);
+
+		long timeLeft = TRAWLER_TIME_LIMIT - Duration.between(startTime, Instant.now()).getSeconds();
+		int minutes = (int) (timeLeft % 3600) / 60;
+		int seconds = (int) timeLeft % 60 > 0 ? (int) timeLeft % 60 : 0;
+
+		String trawlerText = minutes > 0 ?
+				"Time Left: " + minutes + " Mins " + seconds + " Secs" :
+				"Time Left: " + seconds + " Secs";
+
+		// set widget text
```

Comment:
Do we need a comment telling us we're setting the text when the method is literally `.setText()`